scripts/world_bank/datasets/datasets.py

```diff
@@ -277,11 +287,15 @@ def load_json(url, params, response_file):
             return json.load(f)
 
     logging.info("Fetching url %s, params %s", url, params)
-    response = requests.get(url, params=params).json()
-    with open(response_file, 'w') as f:
-        logging.info('Writing response to file %s', response_file)
-        json.dump(response, f, indent=2)
-    return response
+    try:
```

Comment:
If the download fails for existing data, better to abort the processing as autorefresh with partial data will result in deletions in prod.